src/AWS.Logger.Core/AWS.Logger.Core.csproj

```diff
@@ -23,15 +23,15 @@
     <SignAssembly>True</SignAssembly>
     <DelaySign>False</DelaySign>
     <AssemblyOriginatorKeyFile>..\..\awssdk.dll.snk</AssemblyOriginatorKeyFile>
-    <Version>1.6.0</Version>
+    <Version>2.0.0</Version>
   </PropertyGroup>
 
   <PropertyGroup Condition=" '$(TargetFramework)' == 'netstandard1.5' ">
     <DefineConstants>CORECLR</DefineConstants>
   </PropertyGroup>
 
   <ItemGroup>
-    <PackageReference Include="AWSSDK.CloudWatchLogs" Version="3.3.100.39" />
+    <PackageReference Include="AWSSDK.CloudWatchLogs" Version="3.5.0.24" />
```

Comment:
Should this be `3.5.0.2`.  Think that is the minimum version that is using Paginators.